nebula-logger/core/main/logger-engine/classes/Logger.cls

```diff
@@ -2484,40 +2504,55 @@ global with sharing class Logger {
 
     @TestVisible
     private static void setMockScenarioRule(LogScenarioRule__mdt scenarioRule) {
-        SCENARIO_TO_MOCK_SCENARIO_RULE.put(scenarioRule.Scenario__c, scenarioRule);
+        MOCK_SCENARIO_TO_SCENARIO_RULE.put(scenarioRule.Scenario__c, scenarioRule);
     }
 
     /**
      * @description Returns the default save method used when calling saveLog() - set via LoggerSettings__c or by calling setSaveMethod(SaveMethod saveMethod)
      * @return   The enum value of Logger.SaveMethod to use for any calls to saveLog() in the current transaction
      */
     global static SaveMethod getSaveMethod() {
-        return transactionSaveMethod;
+        try {
+            return SaveMethod.valueOf(transactionSaveMethodName);
+        } catch (NoSuchElementException ex) {
+            return null;
```

Comment:
would there be any value in falling back to the event bus for these cases?

Reply:
I've struggled with this part a bit. The problem is that plugins can now provide custom save methods via CMDT (the big object plugin adds 2 custom save methods), which are just `String` values, not actual enum values of `Logger.SaveMethod`. Returning `null` isn't my favorite, but I don't think I want to return the `EVENT_BUS` method since that could be misleading in situations where a plugin is using a custom save method.